README.md

```diff
@@ -1,145 +1,25 @@
-<p align="center">
-  <a href="https://www.airflow.apache.org">
-    <img alt="Airflow" src="https://cwiki.apache.org/confluence/download/attachments/145723561/airflow_transparent.png?api=v2" width="60" />
-  </a>
-</p>
-<h1 align="center">
-  Registry DAG Template
-</h1>
-  <h3 align="center">
-  Guidelines on building, organizing, and publishing example DAGs for the Airflow community to discover and find inspiration from on the <a href="https://registry.astronomer.io/">Astronomer Registry</a>.
-  </br></br>
-  Maintained with ❤️ by Astronomer.
-</h3>
+## Databricks Data Science and Machine Learning Example with Airflow
 
-</br>
+These DAGs give basic examples on how to use Airflow to orchestarte your ML tasks in Databricks. The Databricks code is in a Databricks notebook for which you can find descriptions of below.
 
-<p align="center">
-  For the best user experience by the Airflow community, example DAG repositories should be public on GitHub and follow the structural guidelines and practices presented below.
-</p>
+1. **databricks-automl-example.py** - Runs an end to end data ingest to model publishing pipeline.
+    - **ingest:** Pulls data from BigQuery and does some basic cleaning and transformations then saves it to Delta Lake.
+    - **feature engineering:**  Extract features for model and save output to the Feature Store.
+    - **train:** Train model
+    - **register:** register model to mlflow
 
-</br>
+ ## Requirements
 
-## Before you begin
-An easy way to get started developing and running Apache Airflow pipelines locally is with the Astro CLI. Learn how to install it [here](https://docs.astronomer.io/astro/install-cli).  The CLI is also a great way to take advantage of this repository template.
+ ### Bigquery
+ - [Service account with correct permsissions](https://docs.databricks.com/data/data-sources/google/bigquery.html#step-1-set-up-google-cloud) to use with Databricks cluster
 
-</br>
+ ### Databricks
+ - [Authentication token](https://docs.databricks.com/dev-tools/api/latest/authentication.html) (if you don't want to use a username and password to authenticate from Airflow)
+ - Existing cluster [setup with GCP credentials](https://docs.databricks.com/dev-tools/api/latest/authentication.html) (you can use an on demand cluster but you will need to supply it the GCP credentials accordingly)
```

Comment:
Might be nit-picking, but the docs.databricks URL is mentioned twice in a row here. Could be cleaner by aggregating it. Again, I'm probably nit-picking a little too much.